<ui version="4.0" >
 <author>(C) 2004 Andras Mantia &lt;amantia@kde.org></author>
 <class>UploadProfileDlg</class>
 <widget class="QDialog" name="UploadProfileDlg" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>681</width>
    <height>225</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Upload Profile</string>
  </property>
  <layout class="QGridLayout" >
   <item row="4" column="1" colspan="10" >
    <widget class="QLineEdit" name="linePath" />
   </item>
   <item row="1" column="0" >
    <widget class="QLabel" name="TextLabel4" >
     <property name="text" >
      <string>&amp;Host:</string>
     </property>
     <property name="wordWrap" >
      <bool>false</bool>
     </property>
     <property name="buddy" >
      <cstring>lineHost</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="0" >
    <widget class="QLabel" name="TextLabel2" >
     <property name="text" >
      <string>&amp;User:</string>
     </property>
     <property name="wordWrap" >
      <bool>false</bool>
     </property>
     <property name="buddy" >
      <cstring>lineUser</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="0" >
    <widget class="QLabel" name="TextLabel2_2" >
     <property name="text" >
      <string>Pa&amp;th:</string>
     </property>
     <property name="wordWrap" >
      <bool>false</bool>
     </property>
     <property name="buddy" >
      <cstring>linePath</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2" >
    <widget class="QLabel" name="textLabel1" >
     <property name="text" >
      <string>Profile &amp;name:</string>
     </property>
     <property name="wordWrap" >
      <bool>false</bool>
     </property>
     <property name="buddy" >
      <cstring>lineProfileName</cstring>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="5" >
    <widget class="QCheckBox" name="defaultProfile" >
     <property name="text" >
      <string>Use as &amp;default profile</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="2" >
    <widget class="QLineEdit" name="lineUser" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="MinimumExpanding" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="6" column="9" colspan="2" >
    <widget class="KPushButton" name="cancelButton" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Minimum" hsizetype="Maximum" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize" >
      <size>
       <width>90</width>
       <height>0</height>
      </size>
     </property>
     <property name="text" >
      <string>&amp;Cancel</string>
     </property>
    </widget>
   </item>
   <item row="6" column="7" colspan="2" >
    <widget class="KPushButton" name="okButton" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Maximum" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize" >
      <size>
       <width>90</width>
       <height>0</height>
      </size>
     </property>
     <property name="text" >
      <string>&amp;OK</string>
     </property>
     <property name="autoDefault" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="7" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>445</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1" colspan="3" >
    <widget class="QLineEdit" name="lineHost" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="MinimumExpanding" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="1" column="5" colspan="3" >
    <widget class="QComboBox" name="comboProtocol" />
   </item>
   <item row="1" column="10" >
    <widget class="QLineEdit" name="port" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Maximum" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="1" column="8" colspan="2" >
    <widget class="QLabel" name="TextLabel1_2" >
     <property name="text" >
      <string>Po&amp;rt:</string>
     </property>
     <property name="wordWrap" >
      <bool>false</bool>
     </property>
     <property name="buddy" >
      <cstring>port</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="4" >
    <widget class="QLabel" name="TextLabel1" >
     <property name="text" >
      <string>&amp;Protocol:</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap" >
      <bool>false</bool>
     </property>
     <property name="buddy" >
      <cstring>comboProtocol</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="2" colspan="9" >
    <widget class="KLineEdit" name="lineProfileName" />
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>KPushButton</class>
   <extends>QPushButton</extends>
   <header>kpushbutton.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>lineProfileName</tabstop>
  <tabstop>lineHost</tabstop>
  <tabstop>comboProtocol</tabstop>
  <tabstop>port</tabstop>
  <tabstop>lineUser</tabstop>
  <tabstop>linePath</tabstop>
  <tabstop>defaultProfile</tabstop>
  <tabstop>okButton</tabstop>
  <tabstop>cancelButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>UploadProfileDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>441</x>
     <y>191</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>UploadProfileDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>535</x>
     <y>191</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
